src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -64,46 +65,51 @@
 #include <OutputReportPredefined.hh>
 #include <PlantComponent.hh>
 #include <PlantUtilities.hh>
+#include <Psychrometrics.hh>
 #include <ReportSizingManager.hh>
 #include <UtilityRoutines.hh>
-#include <WaterToWaterHeatPumpEIR.hh>
+#include <PlantLoopHeatPumpEIR.hh>
 #include "CurveManager.hh"
 
 namespace EnergyPlus {
-    namespace EIRWaterToWaterHeatPumps {
+    namespace EIRPlantLoopHeatPumps {
 
-        bool getInputsWWHP(true);
-        std::vector<EIRWaterToWaterHeatPump> eir_wwhp;
-        std::string static const __EQUIP__ = "EIRWaterToWaterHeatPump"; // NOLINT(cert-err58-cpp)
+        bool getInputsPLHP(true);
+        std::vector<EIRPlantLoopHeatPump> heatPumps;
+        std::string static const __EQUIP__ = "EIRPlantLoopHeatPump "; // NOLINT(cert-err58-cpp)
 
-        void EIRWaterToWaterHeatPump::clear_state() {
-            getInputsWWHP = true;
-            eir_wwhp.clear();
+        void EIRPlantLoopHeatPump::clear_state() {
+            getInputsPLHP = true;
+            heatPumps.clear();
```

Comment:
Whoever put in the `eir_wwhp` variable was a real goof.  Snake case in c++, yikes.  This is **much** better :smile: